lib/readline.js

```diff
@@ -1331,7 +1331,8 @@ function codePointAt(str, index) {
   }
   return code;
 }
-exports.codePointAt = codePointAt;
+exports.codePointAt = util.deprecate(codePointAt,
+    'codePointAt() is deprecated. Use String.prototype.codePointAt');
```

Comment:
Instead of deprecating it, could we just alias it to `Function.prototype.call.bind(String.prototype.codePointAt)` and note that it's there for legacy reasons in the docs?